tests/PHPUnit/ConnectionTest.php

```diff
@@ -392,6 +419,44 @@ public function testSetConnectedSitesReturnsEarlyIfNotMultisite() {
 
 		set_connected_blogs( 5, array() );
 	}
+
+	public function testTriggerConnectionRefresh() {
+		M::expectAction( 'airstory_update_all_connections' );
+
+		trigger_connection_refresh( 'foo', 'bar' );
+	}
+
+	public function testTriggerConnectionRefreshReturnsEarlyIfValuesMatch() {
+		$value = uniqid();
+
+		M::userFunction( 'do_action', array(
+			'times' => 0,
+		) );
+
+		trigger_connection_refresh( $value, $value );
+	}
+
+	public function testUpdateAllConnections() {
```

Comment:
Would it be better to test the outcome of the updates rather than just testing that the function was called 4 times?